<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowGestionnaires</class>
 <widget class="QMainWindow" name="MainWindowGestionnaires">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QTabWidget" name="tabWidgetGestionnaire">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Statisitques</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>310</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Personnel</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="tableWidgetEmploye">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>250</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <column>
             <property name="text">
              <string>Nom</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Prenom</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>rue</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>code postal</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ville</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Téléphone</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Téléphone fix</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonModifEmploye">
              <property name="text">
               <string>Modifier la selection</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSupprimer">
              <property name="text">
               <string>Supprimer la selection</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelEmploye">
            <property name="text">
             <string>Ajout d'un employé.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Nom : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditNom"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Prenom :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPrenom"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Pseudo :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPseudo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>rue :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditRue"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Code postal :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditCodePostal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Ville :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditVille"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Tel :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditTel"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Tel fix :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditTelFix"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Type :</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="radioButtonGestionnaire">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Gestionnaire</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonControleur">
                <property name="text">
                 <string>Controleur</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="pushButtonAnnuler">
              <property name="text">
               <string>Annuler</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonValider">
              <property name="text">
               <string>Valider</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Rayon/Type produit</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Rayons </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonSupprimerRayons">
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetRayons"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLineEdit" name="lineEditRayons"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonAjouterRayons">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Ajouter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Catégories</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonSupprimerCategories">
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetCategories"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLineEdit" name="lineEditCategories">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonAjouterCategories">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Ajouter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Produits</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonSupprimerProduits">
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetProduits"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <widget class="QLineEdit" name="lineEditProduits">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonAjouterProduits">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Ajouter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Ajout de produit : </string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Produits demandés :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="listWidgetProduitsDemandes"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QPushButton" name="pushButtonAjouterProduitsDemandes">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Ajouter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonRefuserProduitsDemandes">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Refuser</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Visites</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
